lib/rules/jsx-no-target-blank.js

```diff
@@ -74,6 +74,29 @@ function getStringFromValue(value) {
       if (value.expression.type === 'TemplateLiteral') {
         return value.expression.quasis[0].value.cooked;
       }
+      const expr = value.expression;
+      if (expr.type === 'ConditionalExpression') {
+        if (
+          typeof expr.consequent.value === 'string'
```

Comment:
it kind of seems here like if the consequent is valid, that the alternate can be invalid?

can we add some tests like `condition ? 3 : 'noopener noreferrer'` and `condition ? 'noopener noreferrer' : '3'` etc?

Reply:
Yes, the condition first checks the consequent's value if it's valid then it returns that value, otherwise, it checks the alternate's value if it is valid then it returns that value similar to as in the function `attributeValuePossiblyBlank(attribute)`. 
I thought it was workable since both of the values were not going to be assigned.

Reply:
So should I change the condition for both the branches to be true and then return one of them since both would be true?

Reply:
How about instead of checking for different types of combinations that can be assigned to `rel`  we can just check if it's string type and return it to the `hasSecureRel` function since the other two conditions are doing the same and I think we have to return the string value since that's what the function is doing when called to get the `rel` attributes value. But we have to figure out how to return both the consequent and alternate values?

Reply:
Maybe it could return an array of strings - in every case? That way it'd be an array of 1 string for most nodes, but for a ternary it'd be an array of 2 strings.